src/config/index.js

```diff
@@ -1,40 +1,40 @@
 export const API = Object.freeze({
```

Comment:
Nit: Your editor or prettier config looks like its trying to use 4 spaces instead of 2. Switch it to 2 so we can stay consistent.